AssetManagementTestProject/TestCases/US303LogoutTest.cs

```diff
@@ -0,0 +1,48 @@
+﻿using NUnit.Framework;
+using NUnit.Framework.Internal;
+using AssetManagementTestProject.PageObj;
+using AssetManagementTestProject.TestSetup;
+
+namespace AssetManagementTestProject.TestCase;
+
+[TestFixture]
+public class US303LogoutTest : NUnitWebTestSetup
+{
+    [TestCase(Constant.ADMIN_USERNAME_HN, Constant.BASE_ADMIN_PASSWORD)]
+    [TestCase(Constant.STAFF_USER_NAME, Constant.STAFF_PASSWORD)]
+    public void UserCanLogoutSuccessfully(string userName, string password)
```

Comment:
You can change the method title to `TC01_User...` to know what test case of the US you are working on